app/views/projects/show.html.erb

```diff
@@ -0,0 +1,79 @@
+
+<section class="product-shop spad page-details">
+	<div class="container">
+		<div class="row">
+			<div class="col-lg-9">
+
+				<div class="row">
+
+					<div class="col-lg-6">
+
+						<div class="product-pic-zoom">
+							<img src="project_image.jpg">
+						</div>
+					</div>
+					<div class="col-lg-6">
+						<div class="product-details">
+							<div class="pd-title">
+								<h3> Projet : <%= @project.title.capitalize %> </h3>
+								<br>
+							</div>
+
+							<div class="pd-desc">
+								<% if user_signed_in?%>
```

Comment:
ce n'est pas ce qu'on avait dit, un visiteur non connecté peut tout voir **SAUF** la description longue et le nom des participants

Reply:
je viens de voir que tu avais en fait créé deux blocs pour ça, pour que le code soit plus court tu peux juste mettre le if user sign in sur les participants et la long description pour que ça se répète moins (DRY)